src/pluginmanager.cpp

```diff
@@ -5308,6 +5309,7 @@ void PlugInManager::OnEndPerformCurlDownload(wxCurlEndPerformEvent &ev)
     
     if( m_pCurlThread )
     {
+		m_pCurlThread->Wait();
```

Comment:
I need to make another minor change so please wait at least till 14-10 to merge this PR. 

Best regards,
Dan